### PR TITLE
Reinstate main content div

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Currently the div with [`id="maincontent"`](https://github.com/guardian/frontend/blob/master/common/app/views/main.scala.html#L24) is not being displayed. It appears to have been omitted during the Great Template Fragment Extraction of 2017. This means the ["Skip to main content" link doesn't do anything](https://twitter.com/forzagaribaldi/status/919846654710370304), rendering our pages less accessible.

## What is the value of this and can you measure success?

Reinstates this div, so that clicking the "Skip to main content" link skips past the ads and header.

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Tested in CODE?

No

### CODE REVIEW

**Review by scottp719 — COMMENTED** [review]
👍  make it so

**Comment by ScottPainterGNM** [comment]
(when tests pass)

File: common/app/html/HtmlPage.scala [hunk 1]
```diff
@@ -27,7 +27,16 @@ object HtmlPageHelpers {
...
+    if(model.Page.getContent(page).exists(_.tags.hasSuperStickyBanner))
+      stacked(
+        headerContent,
+        mainContent() when !page.metadata.shouldHideHeaderAndTopAds
+      )
+    else
+      stacked(
+      bannerAndHeaderDiv(headerContent),
+      mainContent() when !page.metadata.shouldHideHeaderAndTopAds
+    )
```
Comment (TBonnin) [inline common/app/html/HtmlPage.scala]: would the following work?
```
...
val header = if(model.Page.getContent(page).exists(_.tags.hasSuperStickyBanner)) headerContent else bannerAndHeaderDiv(headerContent)

stacked(
    header,
    mainContent() when !page.metadata.shouldHideHeaderAndTopAds
)
...
```

**Comment by SiAdcock** [comment]
After some discussion with @NataliaLKB, we decided to move the `<div id="maincontent">` element out of the head and above the body for each page. This makes more sense semantically.

In doing so, I have also deliberately omitted the check that prevents the div from appearing on pages without header ads. The "Skip to main content" link always appears regardless, so it makes sense to always include the `maincontent` div also.

**Comment by NataliaLKB** [comment]
I think we are potentially missing some places like in [liveblogs](https://github.com/guardian/frontend/blob/8721378c33106e4cd7df7ed4581ff9f117fb6903/article/app/pages/LiveBlogHtmlPage.scala) and maybe [articles that aren't a specified kind](https://github.com/guardian/frontend/blob/8721378c33106e4cd7df7ed4581ff9f117fb6903/article/app/pages/ArticleHtmlPage.scala)

**Comment by TBonnin** [comment]
@NataliaLKB that should be covered by `StoryHtmlPage`

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**